build failure

seeing some regression build failure as 

```
error[E0063]: missing fields `icons`, `title` and `website_url` in initializer of `rmcp::model::Implementation`
   --> src/cmd/mcp.rs:366:26
    |
366 |             server_info: Implementation {
    |                          ^^^^^^^^^^^^^^ missing `icons`, `title` and `website_url`

For more information about this error, try `rustc --explain E0063`.
```

- https://github.com/Homebrew/homebrew-core/actions/runs/17676489131/job/50239822985#step:4:362
- https://github.com/Homebrew/homebrew-core/issues/236818

--- a/src/cmd/mcp.rs
+++ b/src/cmd/mcp.rs
@@ -367,7 +367,7 @@ impl ServerHandler for JwtHackServer {
                 name: "jwt-hack".to_string(),
                 version: env!("CARGO_PKG_VERSION").to_string(),
                 title: Some("JWT-HACK".to_string()),
-                website_url: None,
+                website_url: Some(env!("CARGO_PKG_REPOSITORY").to_string()),
                 icons: None,
             },
             instructions: Some("JWT-HACK MCP Server - Provides tools for JWT security testing including decode, encode, verify, crack, and payload generation.".to_string()),

--- a/src/cmd/mcp.rs
+++ b/src/cmd/mcp.rs
@@ -366,6 +366,9 @@ impl ServerHandler for JwtHackServer {
             server_info: Implementation {
                 name: "jwt-hack".to_string(),
                 version: env!("CARGO_PKG_VERSION").to_string(),
+                title: Some("JWT-HACK".to_string()),
+                website_url: None,
+                icons: None,
             },
             instructions: Some("JWT-HACK MCP Server - Provides tools for JWT security testing including decode, encode, verify, crack, and payload generation.".to_string()),
         }

